xm-commons-domainevent-db/src/main/java/com/icthh/xm/commons/domainevent/db/service/DatabaseSourceInterceptor.java

```diff
@@ -0,0 +1,256 @@
+package com.icthh.xm.commons.domainevent.db.service;
+
+import com.icthh.xm.commons.domainevent.config.Column;
+import com.icthh.xm.commons.domainevent.config.EntityFilter;
+import com.icthh.xm.commons.domainevent.config.Query;
+import com.icthh.xm.commons.domainevent.config.SourceConfig;
+import com.icthh.xm.commons.domainevent.config.XmDomainEventConfiguration;
+import com.icthh.xm.commons.domainevent.db.domain.JpaEntityContext;
+import com.icthh.xm.commons.domainevent.domain.DomainEvent;
+import com.icthh.xm.commons.domainevent.service.EventPublisher;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.hibernate.EmptyInterceptor;
+import org.hibernate.metamodel.spi.MetamodelImplementor;
+import org.hibernate.persister.entity.AbstractEntityPersister;
+import org.hibernate.persister.entity.EntityPersister;
+import org.hibernate.type.Type;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.Table;
+import javax.persistence.metamodel.Metamodel;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static com.icthh.xm.commons.domainevent.domain.enums.DefaultDomainEventOperation.CREATE;
+import static com.icthh.xm.commons.domainevent.domain.enums.DefaultDomainEventOperation.DELETE;
+import static com.icthh.xm.commons.domainevent.domain.enums.DefaultDomainEventOperation.UPDATE;
+import static com.icthh.xm.commons.domainevent.domain.enums.DefaultDomainEventSource.DB;
+import static java.lang.Boolean.TRUE;
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class DatabaseSourceInterceptor extends EmptyInterceptor {
```

Comment:
java doc is missng .
Also test is missing for this class.

I suggest add java doc with filter example (for method) isIntercepted.

Also write some test with domainevent.yml with different filters